README.rst

```diff
@@ -0,0 +1,24 @@
+================
+Bitstream Viewer
+================
+
+This tool has two primary functions:
```

Comment:
Please left the original info from README.md:
`The purpose of this tool is to visualize the bits and their corresponding tags in a given bitstream.`

Reply:
I put that description back into the file